designer/client/src/components/graph/node-modal/editors/expression/CustomAceEditorCompleter.tsx

```diff
@@ -30,11 +30,21 @@ function isSpelTokenAllowed(iterator, modeId): boolean {
     return modeId === "ace/mode/spel" || modeId === "ace/mode/spelTemplate";
 }
 
-interface Editor extends Ace.Editor {
-    readonly completer: {
-        activated: boolean;
-        openPopup: (editor: Editor, prefix: string, keepPopupPosition: boolean) => void;
-    };
+declare module "ace-builds" {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Ace {
+        interface Autocomplete {
+            base: Ace.Point;
+            editor: Ace.Editor;
+            activated: boolean;
+            openPopup: (this: Autocomplete, editor: Ace.Editor, prefix: string, keepPopupPosition: boolean) => void;
```

Comment:
Do we need it? looks like it's not called and if i understand correctly, we extend predefined Ace.Autocomplete interface here

Reply:
^